WORKSPACE.bazel

```diff
@@ -146,3 +146,32 @@ http_file(
     sha256 = "c449591174982bbc003d1290003fcbc7b939215436922d2f0f25239d110d531a",
     urls = ["https://repo1.maven.org/maven2/org/jacoco/org.jacoco.cli/0.8.8/org.jacoco.cli-0.8.8-nodeps.jar"],
 )
+
+http_archive(
+    name = "build_bazel_rules_android",
+    sha256 = "cd06d15dd8bb59926e4d65f9003bfc20f9da4b2519985c27e190cddc8b7a7806",
+    strip_prefix = "rules_android-0.1.1",
+    urls = ["https://github.com/bazelbuild/rules_android/archive/v0.1.1.zip"],
+)
+
+load("@build_bazel_rules_android//android:rules.bzl", "android_sdk_repository")
+
+android_sdk_repository(
+    name = "androidsdk",
+    #path = "/usr/local/google/home/cobark/Desktop/Tools/android-sdk",
+    path = "/usr/local/google/home/cobark/Android/Sdk",
```

Comment:
If `path` isn't set, the `android_sdk_repository` rule will look into `ANDROID_HOME`, which is available in CI (see https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#environment-variables-3).